gnat2goto/driver/tree_walk.adb

```diff
@@ -2907,6 +2961,11 @@ package body Tree_Walk is
                                              Rhs => Init_Expr,
                                              Source_Location => Sloc (N)));
       end if;
+
+      if not Is_In_Symtab then
+         Register_Identifier_In_Symbol_Table (Id, Init_Expr);
```

Comment:
I'm not sure I quite understand what what this here does?

Reply:
What this does is that there was a problem where symbols where not added to the symbol table with the values they should have at one path that handled object declarations in the past, and these caused symbol table lookups to fail. This was fixing this.